java/src/com.zeroc.ice/src/main/java/com/zeroc/Ice/Util.java

```diff
@@ -424,8 +423,8 @@ public static String stringVersion() {
     }
 
     /**
-     * Returns the Ice version as an integer in the form <code>A.BB.CC</code>, where <code>A</code>
-     * indicates the major version, <code>BB</code> indicates the minor version, and <code>CC</code>
+     * Returns the Ice version as an integer in the form {@code A.BB.CC}, where {@code A}
+     * indicates the major version, {@code BB} indicates the minor version, and {@code CC}
      * indicates the patch level. For example, for Ice 3.3.1, the returned value is 30301.
      *
      * @return The Ice version.
```

Comment:
This comment is completely incorrect. It returns versions in the form `AABBCC`, not `A.BB.CC`.
To be fair, the comment was already incorrect before this PR.

Reply:
We should also update the example to 3.8.1